src/Controls/src/Core/PointerEventArgs.cs

```diff
@@ -11,15 +11,28 @@ public class PointerEventArgs : EventArgs
 
 		Func<IElement?, Point?>? _getPosition;
 
+		/// <summary>
+		/// Platform-specific arguments associated with the PointerEventArgs
+		/// </summary>
+		public PlatformPointerEventArgs? PlatformArgs { get; private set; }
+
+		/// <summary>
+		/// Public constructor that creates an instance of <see cref="PointerEventArgs"/>
+		/// </summary>
 		public PointerEventArgs()
 		{
 		}
 
-		internal PointerEventArgs(Func<IElement?, Point?>? getPosition)
+		internal PointerEventArgs(Func<IElement?, Point?>? getPosition, PlatformPointerEventArgs? args = null)
 		{
 			_getPosition = getPosition;
+			PlatformArgs = args;
 		}
 
+		/// <summary>
+		/// Returns the position of the pointer relative to the element
```

Comment:
A bit of a nit but I'd use `Gets the position....` instead of `Returns the position...` 
Also don't forget to add a period at the end of _every_ tag.

Also, this is a virtual member, so it needs to have specific wording. Something like `When overridden in a derived class....`  . If no class overrides this method, I be tempted to put the `When overridden...` inside a `<remarks>` tag 

https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Abstract-and-virtual-member